core/sql/executor/ExExeUtil.h

```diff
@@ -4172,6 +4173,48 @@ class ExExeUtilLobInfoPrivateState : public ex_tcb_private_state
 protected:
 };
 
+class connectByStackItem
```

Comment:
I think it is unusual in the executor to have classes that do not have a base class. How about memory management for this class. I might understand more as I see how these classes are used.